charts/airflow/values.yaml

```diff
@@ -328,6 +328,12 @@ airflow:
     ##
     extraVolumes: []
 
+    ## extra containers for the pod template
+    ## - spec for Volume:
+    ##   https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#container-v1-core
+    ##
```

Comment:
This docstring is incorrect, can you please update it to:

```yaml
    ## extra containers for the Pod template
    ## - spec for Container:
    ##   https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#container-v1-core
    ##
```

Reply:
corrected